src/promptflow-core/promptflow/contracts/multimedia.py

```diff
@@ -1,8 +1,9 @@
 import base64
-import filetype
 import hashlib
 from typing import Callable, Optional
 
+import filetype
```

Comment:
why move this import?

Reply:
It is automatically modified by pre-commit.